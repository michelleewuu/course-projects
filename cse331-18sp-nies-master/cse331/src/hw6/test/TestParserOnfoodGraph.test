# Test the parser on food data to see if the graph is built correctly

# Parse through the data and build the graph
LoadGraph food food.tsv

# List nodes and edges to see if the graph is built correctly
ListNodes food
ListChildren food CheeseCake
ListChildren food CarrotCake
ListChildren food FruitTart