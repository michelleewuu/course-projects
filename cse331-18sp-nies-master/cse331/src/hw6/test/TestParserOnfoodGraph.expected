# Test the parser on food data to see if the graph is built correctly

# Parse through the data and build the graph
loaded graph food

# List nodes and edges to see if the graph is built correctly
food contains: CarrotCake CheeseCake FruitTart
the children of CheeseCake in food are: CarrotCake(American Dessert) CarrotCake(Cake) FruitTart(Not Vegan Dessert)
the children of CarrotCake in food are: CheeseCake(American Dessert) CheeseCake(Cake)
the children of FruitTart in food are: CheeseCake(Not Vegan Dessert)